packages/core/src/components/forms/control-group.md

```diff
@@ -1,50 +1,43 @@
-@# Control groups
+@# Control group
 
-A `.@ns-control-group` renders several distinct controls as one unit, squaring the borders between
-them. It supports any number of `.@ns-button`, `.@ns-input`, `.@ns-input-group`, and `.@ns-select`
-elements as direct children.
-
-Note that `.@ns-control-group` does not cascade any modifiers to its children. For example, each
-child must be marked individually as `.@ns-large` for uniform large appearance.
+A control group renders several distinct form controls as one unit, squaring the
+borders between them. It supports any number of buttons, text inputs, input
+groups, and HTML selects as direct children.
 
 <div class="@ns-callout @ns-intent-success @ns-icon-comparison">
     <h4 class="@ns-heading">Control group vs. input group</h4>
     <p>Both components group multiple elements into a single unit, but their usage patterns are
-    different.</p>
-    <p>Think of `.@ns-control-group` as a parent with multiple children, each of them a
-    "control."</p>
-    <p>Conversely, a `.@ns-input-group` is a single control, and should function like so. A
+    quite different.</p>
+    <p>Think of `ControlGroup` as a parent with multiple children, each of them a separate
```

Comment:
The wording of "each of them a separate control" feels a bit awkward. I'd recommend either:
"with each of them..." or
"with each one..."